lib/feature/start/presentation/pages/start_page.dart

```diff
@@ -0,0 +1,207 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:job_interview_practice/feature/question/data/datasources/question_service.dart';
+import 'package:job_interview_practice/feature/question/data/models/question.dart';
+import 'package:job_interview_practice/feature/question/presentation/widgets/appBar.dart';
+
+class StartPage extends StatefulWidget {
+  @override
+  _StartPageState createState() => _StartPageState();
+}
+
+class _StartPageState extends State<StartPage> {
+  // Todo: remove hardcode
+  QuestionService questionService =
+      QuestionService(3, QuestionModel("2", "good", true));
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppBar(),
+      body: Column(
+        children: [
+          SizedBox(
+            height: 100,
+          ),
+          Container(
+            decoration: BoxDecoration(
+              border: Border.all(color: Colors.black54, width: 2),
+              borderRadius: BorderRadius.circular(8),
+            ),
+            child: Padding(
+              padding: const EdgeInsets.all(10.0),
+              child: Column(
+                children: [
+                  Text(
+                      "${questionService.answeredQuestion}/${questionService.numberOfQuestions}"),
+                  Text(
+                    "${questionService.getQuestionText()}",
+                    style: TextStyle(fontSize: 20),
+                  ),
+                ],
+              ),
+            ),
+          ),
+          SizedBox(
+            height: 10,
+          ),
+          Padding(
+            padding: const EdgeInsets.all(100.0),
+            child: Container(
+              child: Image.asset('images/image_interviewer.png'),
+            ),
+          ),
+          Container(
+            padding: EdgeInsets.all(32.0),
+            child: Center(
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: <Widget>[
+                  RaisedButton(
+                    color: Colors.amber,
+                    onPressed: () => Navigator.of(context).pop(),
+                    child: Text('Exit'),
+                  ),
+                  SizedBox(width: 50),
+                  RaisedButton(
+                    color: Colors.green,
+                    // onPressed: () => Navigator.of(context).pop(),
+                    onPressed: () => {
+                      setState(
```

Comment:
again:
in clean architecture, you don't need to use setState any more, whole this stuff should be moved to the bloc. Bloc is responsible for any logic in your page. The page should be a dummy and be responsible only on widgets and how to display them